lib/x11test.pm

```diff
@@ -109,7 +109,7 @@ sub import_pictures {
     };
     assert_screen 'shotwell-importing';
     send_key "ctrl-l";
-    type_string "/home/$username/Documents\n";
+    enter_cmd "/home/$username/Documents";
```

Comment:
IMHO, here the "/home/$username/Documents" is just a string, not a command.
So, change the type_string to enter_cmd might not be applicable for such kind of situation.

Reply:
Right. It will still work though. Also `type_string '…\n'` will work. Maybe the most clean way would be

```suggestion
    type_string "/home/$username/Documents";
    send_key 'ret';
```

WDYT?

Reply:
Keep both type_string and enter_cmd LGTM.